src/components/App/ListWrapper.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+
+interface listData {
+    filmTitle: string
+}
+interface listState {
+    filmData: any,
+    visibleOverview: string,
+    visibleDetailview: string,
+    isActive: string,
+    filmTitle: string
+}
+const listCaption : any = ["Title","Released","Runtime","Genre","Director","imdbRating"];
+
+class ListWrapper extends Component<listData, listState> {
+    _isMounted = false;
+    constructor(props: any) {
+        super(props);
+        this.filmTitle = this.props.filmTitle || '';
+        this.state = {filmData : {}, visibleOverview: "show", visibleDetailview: "hide", isActive: "none"};
+    }
+
+    getData = (filmTitle: string) => {
+        const makeRequestUrl = "http://www.omdbapi.com/?apikey=a96e4d00&t=" + filmTitle;
+        const response = fetch(makeRequestUrl).then((response) => { return response.json(); });
+        response.then((response: any) => {
+            if(this._isMounted) {
+                this.setState({
+                    filmData: response,
+                });
+            }
+        });
+    }
+    componentDidMount() {
+        this._isMounted = true;
+        this.getData(this.filmTitle);
+    }
+
+    componentWillUnmount() {
+        this._isMounted = false;
+    }
+    renderListDom = (filmData: any) => {
+        let listEle : any= [];
+        let movieOverview : any = [];
+        let movieDetailedview : any = [];
+        let listEleLen : number = 0;
+        let processedNodes : number = 0;
+        let visibleOverview = this.state.visibleOverview;
+        let visibleDetailview = this.state.visibleDetailview;
+        let ele = filmData;
+
+        movieOverview = listCaption.map((data: any,idx:number) => {
+            if(typeof(ele[data]) == 'string') {
+                return(<span key={idx}><strong>{data}: </strong>{ele[data]}</span>);
+            } 
+        })
+        
+        for (let keys in ele) {
+            if(typeof(ele[keys]) == 'object') {
+                let obj = ele[keys];
+                let objDom : any = [];
+                obj.forEach((objEle: any) => {
+                    for (let key in objEle) {
+                        objDom.push(<span><strong>{key}: </strong>{objEle[key]}</span>);
+                    }
+                });
+                movieDetailedview.push(<div className="subList"><strong>{keys}: </strong>{objDom}</div>) 
```

Comment:
Missing key attribute.
There is a warning about this in the console.

Reply:
All warnings are cleared.